Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -7234,85 +7299,54 @@ def get_whois_raw(domain, server="", previous=None, rfc3490=True, never_cut=Fals
 
 
 def get_root_server(domain):
-    ext = domain.split(".")[-1]
-    for dble in dble_ext:
-        if domain.endswith(dble):
-            ext = dble
 
-    if ext in list(tlds.keys()):
-        entry = tlds[ext]
-        try:
+    demisto.debug(f"Attempting to get root server from domain '{domain}'...")
+    try:
+        (_, tld) = domain.rsplit(".", 1)
+        for dble in dble_ext:
+            if domain.endswith(dble):
+                tld = dble
+
+        if tld in list(tlds.keys()):
+            entry = tlds[tld]
             host = entry["host"]
-        except KeyError:
-            context = ({
-                outputPaths['domain']: {
-                    'Name': domain,
-                    'Whois': {
-                        'QueryStatus': 'Failed'
-                    }
-                },
```

Comment:
Where is this behavior?

Reply:
This is now added in the `whois_command` when an exception is caught, lines 8661-8668.